plugins/catalog-backend-module-openapi/README.md

```diff
@@ -15,15 +17,18 @@ yarn add --cwd packages/backend @backstage/plugin-catalog-backend-module-openapi
 
 ### Adding the plugin to your `packages/backend`
 
-#### **openApiPlaceholderResolver**
+#### **refPlaceholderResolver**
 
-The placeholder resolver can be added by importing `openApiPlaceholderResolver` in `src/plugins/catalog.ts` in your `backend` package and adding the following.
+The placeholder resolver can be added by importing `refPlaceholderResolver` in `src/plugins/catalog.ts` in your `backend` package and adding the following.
 
 ```ts
-builder.setPlaceholderResolver('openapi', openApiPlaceholderResolver);
+builder.setPlaceholderResolver('openapi', refPlaceholderResolver);
+builder.setPlaceholderResolver('asyncapi', refPlaceholderResolver);
```

Comment:
@jhaals, thanks for the remark! I actually initially had the same idea just to export two functions `asyncApiPlaceholderResolver` and `openApiPlaceholderResolver` pointing to the same implementation. But then I dropped the idea because I figured, that the implementation is not specific to any of the *API specifications but can also be used for other kinds of documents that contain json schema style $refs (e.g., json schema :smile: ).

I also did not want to introduce a placeholder function like `asyncApiPlaceholderResolver` that might right now act just like a `refPlaceholderResolver` but, later on, might evolve to resolve the $refs in a totally different way. For example, `asyncApiPlaceholderResolver` might be evolved to write the referenced schema parts into the components block instead of resolving them inline to preserve the name of the resolved component. This might be a breaking change for installations that already use this resolver because the resulting documents will change.

But I'm open to adding additional aliases `asyncApiPlaceholderResolver` and `openApiPlaceholderResolver` pointing to `refPlaceholderResolver` and documenting that the resulting document might change in the future for those two functions.

Reply:
> It would have been nice to keep the actual validation of the OpenAPI spec but as far as I understand there are discussions about not storing the spec at all in the catalog so maybe all of this will change in the future anyway 🙂

Did the previous method actually do validation as well? In that case I think it would be very good to keep that behaviour for openapi.

Reply:
Thanks for sharing! I see what you mean, the package is not 1.0 so breaking changes are allowed to happen but it's always nice to keep that at a minimum. Would be nice to clarify if validation is done in the previous code as @mfrinnstrom pointed out, if that's the case it might be a good reason to keep these separate.